action/transfer_test.go

```diff
@@ -35,8 +34,6 @@ func TestTransferSignVerify(t *testing.T) {
 	elp, ok := eb.(*envelope)
 	require.True(ok)
 
-	require.Equal("0801100118a08d0622023130522f0a0231301229696f3172633264326465377274757563616c7371763464396e673068323937743633773777766c7068", hex.EncodeToString(elp.serialize()))
-
```

Comment:
suggest to keep `serialize()` and related tests, b/c they help verify protobuf backward compatibility